c10/core/InferenceMode.h

```diff
@@ -61,5 +73,6 @@ struct TORCH_API InferenceMode {
   private:
     bool prev_mode;
     c10::impl::LocalDispatchKeySet prev_keyset;
+    at::AutoGradMode grad_mode;
```

Comment:
nit: prev_grad_mode

Reply:
@albanD Btw `AutoGradMode` is not a state but a RAII guard (it's confusing since we have too many API in grad_mode.h) so it's actually doesn't fit the `prev_` meaning here. ;)